src/main/java/address/MainApp.java

```diff
@@ -48,18 +51,19 @@ public void init() throws Exception {
 
     @Override
     public void start(Stage primaryStage) {
+        LoggerProvider.setLevel(Level.SEVERE);
         setupComponents();
         mainController.start(primaryStage);
 
-        EventManager.getInstance().post(new LoadDataRequestEvent(PreferencesManager.getInstance().getPersonFile()));
+
+        // initial load (requires mainController to be started.
```

Comment:
No closing bracket? Also, can clarify if the double new line above is intentional?